### ISSUE TITLE
[Glyphs] Symbols

### ISSUE THREAD

**Issue by Donnnno** [Arcticons-Team/Arcticons-Font#9]
- [x] Math
- [X] Arrow
- [x] Geometry
- [x] #46 
- [x] #47

**Comment by quantenzitrone** [comment]
are math symbols included in this? like ∂∫∃∈⌀∞∨∧√∇ℤℚℂℕℝ⊷∡⊃⊂∝⊕⊖⊙⊗≤≠≥∩∪ and similar

**Comment by Donnnno** [comment]
@quantenzitrone yes, I've updated the issue to show what has to be done for this!
